packages/optimizer/lib/transformers/ApplyCommonAttributes.js

```diff
@@ -0,0 +1,254 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const parseSizes = require('../parseSizes');
+const {appendChild, createElement, insertText, hasAttribute} = require('../NodeUtils');
+const ID_PREFIX = 'i-amp-';
```

Comment:
The prefix `i-amphtml-` is disallowed, but `i-amp-` is OK. It seems somewhat strange why `i-amp-` wasn't chosen rather for the internal prefix since it has fewer bytes.